engine/flink/avro-util/src/test/scala/pl/touk/nussknacker/engine/avro/schema/FullNameNotCompatible.scala

```diff
@@ -24,7 +24,6 @@ case class FullNameNotCompatible(var first: CharSequence, var middle: CharSequen
       case 0 => first = value.asInstanceOf[CharSequence]
       case 1 => middle = value.asInstanceOf[CharSequence]
       case 2 => last = value.asInstanceOf[CharSequence]
-      case 2 => sex = value.asInstanceOf[CharSequence]
```

Comment:
Why did you remove it?

Reply:
I try to remove compiler warnings when it's possible :)

Reply:
Okey, but field in class still exists? What kinf of warning was there?

Reply:
ok, I understand now :)